src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/request/RequestStatus.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.request;
+
+public enum RequestStatus {
```

Comment:
This should probably be ObjectionStatus.

Reply:
Renamed